docs/headless-cms/06-01-docstopic.md

```diff
@@ -58,6 +60,7 @@ This table lists all possible parameters of a given resource together with their
 | **spec.sources.type** |    **YES**   | Specifies the type of assets included in the DocsTopic CR. |
 | **spec.sources.name** |    **YES**   | Defines an identifier of a given asset. It must be unique if there is more than one asset of a given type in a DocsTopic CR. |
 | **spec.sources.mode** |    **YES**   | Specifies if the asset consists of one file or a set of compressed files in the ZIP or TAR format. Use `single` for one file and `package` for a set of files.  |
+| **spec.sources.metadata** |    **NO**   | Specifies a set of metadata for the Asset. Define it in a valid YAML or JSON format. |
```

Comment:
in each doc where we describe **spec.sources.metadata** specify for the user that one of the possible use case for metadata is to use is as a contract between source and the presentation layer, to enable UI to decide on a  source level, if something specific has to be done/modified in the source before it is rendered for the user